install_virtualenv.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+virtualenv_path=$1
```

Comment:
should we add a check that $1 is not empty?

Reply:
sure that's a good idea

Reply:
sweet